frontend/app-development/features/textEditor/TextEditor.tsx

```diff
@@ -28,7 +28,7 @@ interface TextEditorProps extends React.PropsWithChildren<any> {
 }
 const storageGroupName = 'textEditorStorage';
 export const TextEditor = ({ language }: TextEditorProps) => {
-  const [searchParams, setSearchParams] = useSearchParams();
+  const [searchParams, setSearchParams] = useSearchParams({ lang: defaultLangCode });
```

Comment:
Denne feilen kunne vært unngått om vi hadde hatt en type for forventet searchParams.  :D

```
type TextEditorSearchParams = {
  lang: LangCode;
}
const [searchParams, setSearchParams] = useSearchParams<TextEditorSearchParams>({ lang: defaultLangCode });
```